containers/jetty-http/src/main/java/module-info.txt

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I use module-info.txt when it is not possible for some reason to modularize a module but some work was already done and should not be lost. A module could be modularised later (might be due to some not modularized dependencies or any other reason) and it would be easier to start just by renaming module-info.txt into module-info.java.